tests/utils/test_scatter.py

```diff
@@ -1,39 +1,97 @@
 import mlx.core as mx
+import pytest
 
-from mlx_graphs.utils import scatter
+from mlx_graphs.utils import scatter, degree
 
 
-def test_scatter():
-    src = mx.array([1.0, 1.0, 1.0, 1.0])
-    index = mx.array([0, 0, 1, 2])
-    num_nodes = src.shape[0]
-
-    y_hat1 = scatter(src, index, aggr="softmax")
+@pytest.mark.parametrize(
```

Comment:
The thing with parametrized tests is that if a test fails, it doesn't show which parameters failed.

Reply:
You should be able to see it by running pytest in verbose mode (`pytest -v`)